client/components/MorphologyViewer/index.js

```diff
@@ -67,27 +66,26 @@ class MorphologyContainer extends React.Component {
       this.shouldRender();
     }
   }
-  shouldRender() {
-    // toggle rendering
-  }
-  async addBrainMesh () {
-
-  }
-  makeVisualizer() {
+  async makeVisualizer() {
     let { morphoData } = this.state;
-    let { name, wholeBrain, staticContentLocation } = this.props;
-    const wholeBrainMeshURL = staticContentLocation + "/brain-mesh"
+    let { name, wholeBrain, token } = this.props;
     if (this.viewContainer && morphoData) {
       try {
         this.viewer = new morphoviewer.MorphoViewer(this.viewContainer);
         if (wholeBrain) {
-          this.viewer.addStlToMeshCollection(
+          const wholeBrainMeshURL =
+            "https://bbp.epfl.ch/nexus/v0/data/bbp/atlas/brainparcellationmesh/v0.1.0/39a3078c-1e21-4e1c-9015-82a10ba6797e/attachment";
```

Comment:
hardcoded URL, is it normal?

Reply:
There's only one possible dataset to load at the moment. In the far future there may be more...

Reply:
Actually, these things should be resolved in the data somehow, this is a good question for @annakristinkaufmann and @MFSY